render_event_consts.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"strings"
+)
+
+func (renderer *Renderer) Render_event_consts() ([]byte, error) {
+	f := NewLineWriter()
+	f.WriteLine(`// GENERATED FILE: DO NOT EDIT!`)
+	f.WriteLine(``)
+	f.WriteLine(`package ` + renderer.Package)
+	var constants []string
+	for _, namedAny := range renderer.Model.ComponentsSpecificationExtension {
+		constants = append(constants, namedAny.Value.Yaml)
+	}
+	f.WriteLine(`const (`)
+	//generate constants FactStreamType
+	if len(constants) > 0 {
+		for _, constant := range splitLineConstants(constants[0]) {
+			f.WriteLine(constant + `FactStreamType` + `=` + `"` + constant + `_facts"`)
+		}
+	}
+	f.WriteLine(`)`)
+	f.WriteLine(`const (`)
+	//generate constants CommandsStreamType
+	if len(constants) >= 1 {
+		for _, constant := range splitLineConstants(constants[1]) {
+			f.WriteLine(constant + `CommandsStreamType` + `=` + `"` + constant + `_commands"`)
+		}
+	}
+	f.WriteLine(`)`)
+	return f.Bytes(), nil
+}
```

Comment:
please `go fmt` your files

Reply:
it was done, maybe it doesn't work as you would like.
What needs to be fixed?
Added two blank lines.